python/nav/web/templates/base.html

```diff
@@ -15,7 +15,6 @@
     <link rel="stylesheet" href="{{ STATIC_URL }}css/nav/custom.css" />
     <link rel="stylesheet" href="{{ STATIC_URL }}css/nav/calendar.css" />
     <link rel="stylesheet" href="{{ STATIC_URL }}css/nav/treeselect.css" />
```

Comment:
A good question for the future (not this PR) might be why these are included as multiple style sheet files, and not as part of `nav.css` ...